-------------------------------------------------------------------------------
Test set: io.harness.aida.AppTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.039 s <<< FAILURE! - in io.harness.aida.AppTest
shouldAnswerWithTrue(io.harness.aida.AppTest)  Time elapsed: 0.007 s  <<< FAILURE!
java.lang.AssertionError: 
IndexOutOfBoundsException was thrown.
error:

Index 3 out of bounds for length 3
	at io.harness.aida.AppTest.shouldAnswerWithTrue(AppTest.java:35)

